<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_ngrams_per_cluster</class>
 <widget class="QWidget" name="widget_ngrams_per_cluster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_graph">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>90</y>
     <width>801</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>185</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hlayout_radio">
    <item>
     <widget class="QRadioButton" name="radio_hour">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_day_week">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>day of week</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>221</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="hlayout_clusters" stretch="1,6">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cluster:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_clusters"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlayout_ngrams" stretch="1,6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>N-grams:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_ngrams"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
